setup.py

```diff
@@ -10,7 +10,7 @@
     "aiosqlite==0.13.0",  # asyncio wrapper for sqlite, to store blocks
     "aiohttp==3.6.2",  # HTTP server for full node rpc
     "colorlog==4.1.0",  # Adds color to logs
-    "chiavdf==0.12.16",  # timelord and vdf verification
+    "chiavdf==0.3b11",  # timelord and vdf verification
```

Comment:
Why the strange version number?

Reply:
This is how I used to tag new_code branch (which this code depends on):
https://github.com/Chia-Network/chiavdf/pull/20
We can use normal version numbers once it's merged into master.